Patterns/English/English-DateTime.yaml

```diff
@@ -108,6 +108,9 @@ RelativeMonthRegex: !nestedRegex
   references: [RelativeRegex]
 MonthRegex: !simpleRegex
   def: \b(?<month>apr(il)?|aug(ust)?|dec(ember)?|feb(ruary)?|jan(uary)?|july?|june?|mar(ch)?|may|nov(ember)?|oct(ober)?|sept(ember)?|sep)(?!\p{L})
+# handling cases like 29Feb
```

Comment:
I'd change the comment to: To be used only to handle informal cases like 29Feb. For all other date cases, use MonthRegex.